query-translator/tests/translate.rs

```diff
@@ -49,48 +49,63 @@ fn translate<T: Into<Option<u64>>>(schema: &Schema, input: &'static str, limit:
     select.query.to_sql_query().unwrap()
 }
 
-#[test]
-fn test_coll() {
+fn prepopulated_schema() -> Schema {
```

Comment:
Thanks, I was going to request this at various times and figured you'd get there at some point.  You did!